src/dune_file.ml

```diff
@@ -678,6 +678,7 @@ module Buildable = struct
     ; ocamlopt_flags           : Ordered_set_lang.Unexpanded.t
     ; js_of_ocaml              : Js_of_ocaml.t
     ; allow_overlapping_dependencies : bool
+    ; ocamlformat : bool
```

Comment:
Should this be a property of libs and binary or just of the current directory ?
also, You might want to be able to add exceptions, useful for automatically generated source files

Reply:
it's now possible to use OSL to perform exclusions.
per directory would work, but we already have a mapping between files and modules in buildables, so that seems to be the correct abstraction (at least it's the simplest one to start)